### PR TITLE
APUB-342: Showcase User Workload Monitoring

### PR DESCRIPTION
**Pull request by akosma**
Adds a new tutorial to the APPUiO Cloud documentation, showing how to configure monitoring (Prometheus, AlertManager) to provide live alerts through [Pushover](https://pushover.net/).

This PR also extends the AlertManager configuration page with information about how to configure Pushover.

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `how-to`, `tutorial`, `reference`, `explanation`, `cicd`, `dependency`
      as they show up in the changelog
- [ ] Link this PR to related issues if applicable.

### CODE REVIEW

**Comment by tobru** [comment]
Very cool, thanks :+1:

**Review by tobru — APPROVED** [review]

File: docs/modules/ROOT/pages/tutorials/monitoring/index.adoc [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+.. The Pushover user key and API keys, as shown on the Pushover user interface.
++
+[source]
+--
+APPUIO_CLOUD_ZONE=cloudscale-lpg-2     (or exoscale-ch-gva-2-0)
+APP_PROJECT=xxxxx-fortune
+GRAFANA_PROJECT=xxxxx-grafana
+PUSHOVER_USER_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+PUSHOVER_API_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+--
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/tutorials/monitoring/index.adoc:65]:
```suggestion
APPUIO_CLOUD_ZONE=cloudscale-lpg-2 <1>
APP_PROJECT=xxxxx-fortune
GRAFANA_PROJECT=xxxxx-grafana
PUSHOVER_USER_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
PUSHOVER_API_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
--
<1> You can use other APPUiO Cloud Zones too. See https://portal.appuio.cloud/zones for a list of alternatives.
```

File: docs/modules/ROOT/pages/tutorials/monitoring/index.adoc [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+. During the interaction with the application, the Alert screen on the APPUiO Cloud console will display a message indicating that the alert has been triggered.
++
+image::monitoring/tutorial/console-monitoring-alerts-firing.png[]
+
+. The Pushover interface should show that an alert arrived, and the Pushover client should show the notification, including the error message specified in the YAML file.
++
+image::monitoring/tutorial/pushover-view-application.png[]
++
+image::monitoring/tutorial/pushover-web-client.png[]
+
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/tutorials/monitoring/index.adoc:113]: Please add a link to the troubleshooting section at https://docs.appuio.cloud/user/how-to/configure-alertmanager.html#troubleshooting.

File: docs/modules/ROOT/pages/tutorials/monitoring/index.adoc [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+.. Select "Prometheus" (first entry on the list)
+.. Follow the xref:how-to/use-custom-grafana.adoc[instructions] to configure Prometheus as a data source.
+. Get the values of the token and certificates from the deployment.
++
+[source,bash]
+--
+TOKEN=$(oc get secret $GRAFANA_PROJECT-viewer -ojsonpath={.data.token} | base64 -d)
+CA=$(oc get secret $GRAFANA_PROJECT-viewer -ojsonpath={.data."service-ca\.crt"} | base64 -d)
+echo $CA
+echo "Bearer $TOKEN"
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/tutorials/monitoring/index.adoc:139]:
```suggestion
# Your Certificat Authority (CA)
oc get secret $GRAFANA_PROJECT-viewer -ojsonpath={.data."service-ca\.crt"} | base64 -d
# Your Bearer HTTP access token
echo "Bearer $(oc get secret $GRAFANA_PROJECT-viewer -ojsonpath={.data.token} | base64 -d)"
```

**Review by corvus-ch — APPROVED** [review]
Many thanks.

I left some suggestions but do not have any strong feeling if you address them or not.

File: docs/modules/ROOT/pages/tutorials/monitoring/index.adoc (hunk 2 above)
Comment (akosma) [reply docs/modules/ROOT/pages/tutorials/monitoring/index.adoc:114]: Done

**Pull request merged**